doc/manual/src/advanced-topics/distributed-builds.md

```diff
@@ -156,3 +156,19 @@ option `builders-use-substitutes` in your local `nix.conf`.
 
 To build only on remote builders and disable building on the local
 machine, you can use the option `--max-jobs 0`.
+
+## Troubleshooting
+
+### ssh
+
+even if the store is reachable with `nix store ping --store your_store_url`, the nix-daemon might not be able to reach it. nix-daemon operates as root so you need to make sure that the machine is reachable via ssh as root user.
+
+- add the `-vvv` flags to show more information on the potential problem.
+- make sure your store is reachable by the root user. `sudo ssh -i "your_ssh_key_path" your_user@your_hostname`
+- make sure your builders configuration contains the ssh_key, user, and hostName
```

Comment:
Why not configure the user and key via ssh_config?

Reply:
those need to be valid for the root user. ssh_config is for your user, if I'm not wrong.

Reply:
ssh_config is for every user, you just need to set it for the root user which is way more convenient than fiddling with ssh options around.

Reply:
how about if I changed that sentence to
`- make sure your ssh_config is set for the root user to access that server or that your builders configuration contains the ssh_key, user and hostname`